scipy/stats/tests/test_stats.py

```diff
@@ -7550,6 +7550,14 @@ def test_pvalue_literature(self):
         _, pvalue, _ = stats.multiscale_graphcorr(x, y, random_state=1)
         assert_allclose(pvalue, 1/1001)
 
+    @pytest.mark.slow
+    @pytest.mark.parametrize("is_twosamp", [True, False])
+    def test_alias(self, is_twosamp):
+        x = np.arange(100)
+        y = np.arange(100)
+        res = stats.multiscale_graphcorr(x, y, is_twosamp=is_twosamp)
+        assert_equal(res.stat, res.statistic)
```

Comment:
Do we need a new test or can we tack this on to and existing test?

Reply:
I didn't spot an obvious candidate. On reflection, maybe it would be better to add a `test_result_attributes` method similar to `TestAndersonKSamp` and some other test classes in `test_morestats`.

Reply:
That's fine, too, although if the tests really are slow, it might be better to limit unnecessary number crunching.
I suppose the intent of the comment is to target the test at the changes in the PR. Currently, this test introduces new data and is `@parametrize`d over a keyword argument that is unlikely to affect the outcome of the test. I agree that `is_twosamp` should have been tested, but it's easier to get PRs merged if their scope is focused. If you are interested in improving the `multiscale_graphcorr` test suite more generally, a separate PR for that would be preferable.

Reply:
OK, I've changed the test data to be more consistent with the other tests and removed the parameterisation. If you prefer, I could merge the test with e.g. `test_pvalue_literature`, but since the tests are testing for different things (numerical validity of results vs. result object API) it may make more sense to keep them seperate.